addons/website_event_exhibitor/security/security.xml

```diff
@@ -4,7 +4,7 @@
     <record id="event_sponsor_rule_share" model="ir.rule">
         <field name="name">Event Sponsor: public/portal sponsor or published only</field>
         <field name="model_id" ref="website_event_exhibitor.model_event_sponsor"/>
-        <field name="domain_force">['|', ('exhibitor_type', '=', 'sponsor'), ('website_published', '=', True)]</field>
+        <field name="domain_force">[('website_published', '=', True)]</field>
```

Comment:
I thought that rule was automagically added by the "website.published.mixin" ?

Is it not the case or do you want something specific here?

Reply:
I don't think published mixin implies anything from ACLs point of view... there is no concept of "mixin" or "generic" ACLs. I think rules are still up to final model choice.